scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,147 @@
+@import 'settings';
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
+    position: relative;
+
+    .p-equal-height-row__col {
+      border-top-color: $colors--theme--border-default;
+      border-top-style: solid;
+      border-top-width: 1px;
+      display: grid;
+      grid-column: span $grid-columns-small;
+      grid-row: span 4;
+      grid-template-rows: subgrid;
+      position: relative;
+
+      .p-equal-height-row__col-item {
+        grid-column: span $grid-columns-small;
+      }
+
+      .p-equal-height-row__col-item:first-child {
+        grid-row: auto;
+      }
+
+      @media screen and ($breakpoint-small <= width < $breakpoint-large) {
+        grid-column: span $grid-columns-medium;
+        grid-template-columns: subgrid;
+        margin-top: -1px;
+
+        .p-equal-height-row__col-item {
+          grid-column: span calc($grid-columns-medium / 2);
+        }
+
+        .p-equal-height-row__col-item:first-child {
+          grid-row: span 100;
+        }
+      }
+
+      @media screen and (width >= $breakpoint-large) {
+        border: none;
+        grid-column: span calc($grid-columns / 4);
+      }
+    }
+
+    // divider styles
+    // remove row level dividers for small screen
+    &[class*='p-equal-height-row--divider-below-item-']::before,
```

Comment:
I guess this was renamed together with `__col` child?

I would leave this as `has-divider-…`. When multiple modifiers are possible on a component (like multiple dividers here), we use `is-` or `has-` classes.

Reply:
Yes, I renamed it and removed `has-` as it's getting verbose, I can put it back, but I was wondering if you have any ideas on how to shorten this?

Reply:
`has-divider-1`, etc is less verbose, but also less clear what that does. 

Maybe: `has-item-2-divider`, `has-item-3-divider` that would place a divider above that numbered item? - but again, by making it shorter we are losing a bit of clarity

Reply:
That last one sounds good :slightly_smiling_face: